scripts/backups.sh

```diff
@@ -17,24 +17,20 @@ function s3_config() {
 
 # Cleanup S3 bucket
 function clean_s3bucket() {
-  S3_BUCKET=$1
-  DEL_DAYS=$2
-  if [[ $(s3cmd ls s3://${BUCKET} 2>&1 | grep -q 'NoSuchBucket' ) ]];then
```

Comment:
Why delete this check?. It's just a sanity check to make sure that clean_buckets only works on buckets that exists